src/rich-text-editor.component.js

```diff
@@ -147,7 +147,7 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
 
       return () => styleElement.parentNode.removeChild(styleElement)
     }
-  },[props.placeholder, props.placeholderColor, bandicootId.current])
+  },[props.placeholder, props.placeholderStyling, bandicootId.current])
```

Comment:
Instead of a string of css, it is more common and well understood to pass in a styles object. I think a `placeholderStyle` prop would make more sense here.

Reply:
agreed, but then I need logic for things like `backgroundColor` -> `background-color`
`fontSize` in my case